### ISSUE TITLE
[regression] Shared libraries lost SONAMEs in 2018_03_2

### ISSUE THREAD

**Issue by yurivict** [rdkit/rdkit#1923]
For example, 2018_03_1 installed

```
lib/libRDKitAlignment.so
lib/libRDKitAlignment.so.1
lib/libRDKitAlignment.so.1.2018.03.2
lib/libRDKitCatalogs.so
lib/libRDKitCatalogs.so.1
lib/libRDKitCatalogs.so.1.2018.03.2
...
```

now 2018_03_2 installs only
```
lib/libAlignment.so
lib/libCatalogs.so
...
```

SONAMEs were, presumably, ```libRDKitAlignment.so.1``` and ```libRDKitCatalogs.so.1```, now they are ```libRDKitAlignment.so``` and ```libRDKitCatalogs.so```.

SONAMEs are needed to control when interface changes require depending packages to be rebuilt. "1" should be changed to "2", etc. when such interface changes are made. Otherwise, without SONAMEs, it becomes error-prone. Depending packages will not be rebuilt and they will be randomly crashing.

**Comment by yurivict** [comment]
I can't update the FreeBSD package because of this problem.
The plist in 2018_03_1 made sense, and now, without SONAMEs, it doesn't.

**Comment by greglandrum** [comment]
Yes, this was fixed in #1915

**Comment by yurivict** [comment]
Thanks!
